### ISSUE TITLE
[Designer][Bug 40020451](https://microsoft.visualstudio.com/OS/_workitems/edit/40020451): [ShowCardWizard.JSON]: Screen reader does not announce collapse/expand state for buttons "1.Name", "2.Address", "3.Phone".

### ISSUE THREAD

**Issue by aaagrawal** [microsoft/AdaptiveCards#7609]
### Problem Description

[Bug 40020451](https://microsoft.visualstudio.com/OS/_workitems/edit/40020451): [ShowCardWizard.JSON]: Screen reader does not announce collapse/expand state for buttons "1.Name", "2.Address", "3.Phone".

### Card JSON

```JSON
[Bug 40020451](https://microsoft.visualstudio.com/OS/_workitems/edit/40020451): [ShowCardWizard.JSON]: Screen reader does not announce collapse/expand state for buttons "1.Name", "2.Address", "3.Phone".
```

**Comment by vagpt** [comment]
This issue is still repro at our end.

**Comment by Rishikesh6521** [comment]
Already this bug has been logged and tracking the bug through #8976
